kafka/pom.xml

```diff
@@ -34,6 +34,7 @@
     <description>Kafka interface for Adaptive Alerting.</description>
 
     <properties>
+        <main.basedir>${project.parent.basedir}</main.basedir>
```

Comment:
you mayn't need this config change. similarly in other module's pom as well.

Reply:
Understood.